NOTES.md

```diff
@@ -1,3 +1,24 @@
+# Release 1.16.1 (Stand By Me)
+
+Released on 2015-02-12
+
+Bug fix update.
+
+* Now write to the log every time a file is deleted (#1780)
+* Fixed `rethinkdb dump` and other backup scripts to correctly detect the server version (#3706)
+* Changed the output of `rethinkdb dump` to clarify that indexes are being saved (#3708)
+* Fixed memory consumption with newer versions of boost (#3712)
+* Fixed the `written_docs_total` field of `rethinkdb.stats` (#3713)
+* Fixed a bug that caused the web UI to hang when creating indexes (#3718)
+* Fixed how secondary index status was being reported (#3721)
```

Comment:
I would merge this with 3718 into
"Fixed a bug that caused the web UI to hang when creating or deleting indexes (#3717, #3721)"